packages/cli/src/__tests__/ceramic-daemon.test.ts

```diff
@@ -161,6 +158,32 @@ describe('Ceramic interop: core <> http-client', () => {
         expect(DoctypeUtils.serializeState(doc3.state)).toEqual(DoctypeUtils.serializeState(doc4.state))
     })
 
+    // TODO Enable this test when its possible to use 'sync' flag to disable querying pubsub entirely
```

Comment:
This test will become possible to write correctly as part of https://github.com/ceramicnetwork/js-ceramic/issues/1123, which will make it possible to load a doc without querying pubsub at all.  Right now `sync` only controls whether or not we wait for a reply, but we always do the query, which introduces a race condition around how fast the response comes back and if it comes back before we've built the Doctype instance and taken the current version of the state from the RunningState

Reply:
I was wrong about why this test was failing, and was able to get it working